soh/src/code/z_player_lib.c

```diff
@@ -412,6 +412,7 @@ void func_8008EC70(Player* this) {
 
 void Player_SetEquipmentData(GlobalContext* globalCtx, Player* this) {
     if (this->csMode != 0x56) {
+        this->currentMask = CVar_GetS32("gMaskNum", 0);
```

Comment:
Now that we have proper versioned save support you can add this to the save directly.